app/src/main/res/layout/main_fragment.xml

```diff
@@ -11,7 +11,7 @@
         android:id="@+id/actStock"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:hint="Enter stock"
+        android:hint="@string/actStock_Text"
```

Comment:
changed hard-coded "enter stock" in the text box to a value. This will help if the textbox wants to be duplicated to allow the user to track multiple stocks.